tests/gdimagecopymerge/gdimagecopymerge.c

```diff
@@ -0,0 +1,41 @@
+#include "gd.h"
```

Comment:
please add comment blocks to all new files that give a brief overview of the file and its goals/purposes.  i know our existing files haven't been great in this regard, but we should be doing it better with new ones.

Reply:
totally agree